tariff/stekker.go

```diff
@@ -59,18 +61,27 @@ func NewStekkerFromConfig(other map[string]interface{}) (api.Tariff, error) {
 		return nil, err
 	}
 
+	interval := time.Hour
+
 	switch cc.Region {
 	case "BE":
 		cc.Region = "BE-900"
 	case "NL":
 		cc.Region = "NL-900"
 	}
 
+	if idx := strings.LastIndex(cc.Region, "-"); idx != -1 {
```

Comment:
Why is any of this needed? If only BE/NL have 900s just set interval in the switch above?